examples/listbox/listbox-grouped.html

```diff
@@ -0,0 +1,243 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8" />
+<title>Grouped Option Listbox Example | WAI-ARIA Authoring Practices 1.1</title>
+
+<!--  Core js and css shared by all examples; do not modify when using this template. -->
+<link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+<link rel="stylesheet" href="../css/core.css">
+<script src="../js/examples.js"></script>
+<script src="../js/highlight.pack.js"></script>
+<script src="../js/app.js"></script>
+<script src="../js/utils.js" type="text/javascript"></script>
+
+<link href="css/listbox.css" rel="stylesheet">
+<script src="js/listbox.js" type="text/javascript"></script>
+<script src="js/listbox-scrollable.js" type="text/javascript"></script>
+</head>
+<body>
+  <nav aria-label="Related Links" class="feedback">
+    <ul>
+      <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/projects/13">Related Issues</a></li>
+      <li><a href="../../#Listbox">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+  <h1>Listbox Example with Grouped Options</h1>
+  <p>
+    The following example implementation of the <a href="../../#Listbox">design pattern for listbox</a> demonstrates a single-select listbox widget with grouped options.
+    This widget is functionally similar to an HTML <code>select</code> element with labelled <code>optgroup</code> elements used to group options.
```

Comment:
Would it be worth to include something like “…similar to an HTML <code>select</code> element with the `multiple` attribute set that contains labelled <code>optgroup</code> elements…”? But perhaps with less awkward wording than my pre coffee comment. Not sure if this is necessary tho.

Reply:
I think it's still only single-select; maybe "similar to an HTML `select` element with `size` > 1 ..." would work?

Reply:
Ah yes of course. Thanks for the addition!